content/en/docs/tasks/administer-cluster/kms-provider.md

```diff
@@ -31,6 +41,17 @@ as the Kubernetes control plane, is responsible for all communication with the r
 To configure a KMS provider on the API server, include a provider of type `kms` in the
 `providers` array in the encryption configuration file and set the following properties:
 
+### KMS v1
+* `name`: Display name of the KMS plugin.
+* `endpoint`: Listen address of the gRPC server (KMS plugin). The endpoint is a UNIX domain socket.
```

Comment:
Probably worth documenting how to configure the use of an abstract UDS.

Reply:
When discussing with Trousseau's users, often, this is not understood that the socket will be created on the control plane due to the proximity requirements with the api server processes. 
That might be worth mentioning too.

Reply:
added https://github.com/kubernetes/website/pull/35046/commits/5cdb55d2eb4d9652d46fa1e50a53d712ad8745e7